config.md

```diff
@@ -34,6 +34,37 @@ Each container has exactly one *root filesystem*, specified in the *root* object
 }
 ```
 
+## Mount Points
+
+You can add array of mount points inside container as `mounts`.
+Each record in this array must have configuration in [runtime config](runtime-config.md#mount-configuration).
+
+* **name** (string, required) Name of mount point. Used for config lookup.
+* **path** (string, required) Destination of mount point: path inside container.
```

Comment:
`<nit>`Maybe “Mount destination in the container filesystem”?  The colon-separated form works, but it took me a reread to understand what it meant.`</nit>`.